calculator-app/src/Components/Buttons.css

```diff
@@ -0,0 +1,30 @@
+.all-buttons {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  width: 5px;
+}
+
+.calc-background {
+  padding-bottom: 10px;
+  border-radius: 5px;
+}
+
+.zero {
+  width: 345px;
+}
+
+button {
```

Comment:
It's better practice to create a separate classname vs using the tag name. This effects all buttons on the website vs just the buttons you wanted to add styling to